tool/configure.dart

```diff
@@ -477,6 +477,7 @@ import 'package:cw_core/wallet_base.dart';
 import 'package:cw_core/wallet_credentials.dart';
 import 'package:cw_core/wallet_info.dart';
 import 'package:cw_core/wallet_service.dart';
+import 'package:eth_sig_util/util/utils.dart';
```

Comment:
**Minor:**
This is unused, right?

Reply:
No it's been used. The bytesToHex function comes from it